torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -3355,6 +3355,12 @@ std::vector<int64_t> _output_size(
   return ret;
 }
 
+// return true if v is a real float
+// and false if it is an integer
+bool _is_float_value(float v) {
```

Comment:
Same comment here: this is suspicious.

Reply:
Also TorchScript / Python you a `double` for all floating point values so this should be using `double` as well.